### DISCUSSION TITLE
[PSD7] Exercise 2.14

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#157] [Esercizi (A-L)]
![image](https://user-images.githubusercontent.com/69389461/210018712-0c0ddd02-fe4e-42d5-837c-a975da67a37c.png)

**Comment by Elia-Belli** [comment]
- (a) $Y = \bar A B (C + \bar C) = \bar A B$
- (b) $Y = \overline {A B C} + A \bar B = \bar A  + \bar B + \bar C + A \bar B = \bar A + \bar B + \bar C$
- (c) $Y = ABC\bar D + A \overline {BCD} + (\overline {A+B+C+D}) =$
   $ABC\bar D + A (\bar B + \bar C + \bar D)+(\bar A \bar B \bar C \bar D) = ABC\bar D + A\bar B + A\bar C + A \bar D + \bar A \bar B \bar C \bar D =$
$A\bar D + \bar B(A + \bar A \bar C \bar D) + A\bar C = A \bar D + A \bar C + A\bar B + \bar B \bar C \bar D$

**Comment by fraxb04** [comment]
corretto

**Comment by FeddyLix17** [comment]
Le soluzioni di @Elia-Belli sono corrette, integro il tutto anche con le tabelle di verità e le k-map di ogni caso per verificare la correttezza delle sue soluzioni (e come anche richiesto dall'esercizio)

_Tabella verità (a)_
|A|B|C|Y| 
|-|-|-|-|
|0|0|0|0|
|0|0|1|0|
|0|1|0|1|
|0|1|1|1|
|1|0|0|0|
|1|0|1|0|
|1|1|0|0|
|1|1|1|0|

_K-map (a)_
|AB\C|0|1| 
|-|-|-|
|00|0|0|
|01|1|1|
|11|0|0|
|10|0|0|

_Tabella verità (b)_
|A|B|C|Y| 
|-|-|-|-|
|0|0|0|1|
|0|0|1|1|
|0|1|0|1|
|0|1|1|1|
|1|0|0|1|
|1|0|1|1|
|1|1|0|1|
|1|1|1|0|

_K-map (b)_
|AB\C|0|1| 
|-|-|-|
|00|1|1|
|01|1|1|
|11|1|0|
|10|1|1|

_Tabella verità (c)_
|A|B|C|D|Y| 
|-|-|-|-|-|
|0|0|0|0|1|
|0|0|0|1|0|
|0|0|1|0|0|
|0|0|1|1|0|
|0|1|0|0|0|
|0|1|0|1|0|
|0|1|1|0|0|
|0|1|1|1|0|
|1|0|0|0|1|
|1|0|0|1|1|
|1|0|1|0|1|
|1|0|1|1|1|
|1|1|0|0|1|
|1|1|0|1|1|
|1|1|1|0|1|
|1|1|1|1|0|

_K-map (c)_
|AB\CD|00|01|11|10| 
|-|-|-|-|-|
|00|1|0|0|0|
|01|0|0|0|0|
|11|1|1|0|1|
|10|1|1|1|1|
